openapi/mx_platform_api.yml

```diff
@@ -791,6 +791,11 @@ components:
           example: chase
           nullable: true
           type: string
+        instructional_text:
+          example: Some instructional text <a href="https://example.url.chase.com/instructions"
+            id="instructional_text" >for end users</a>.
```

Comment:
It looks like the space got mixed up. I think we want `id="instructional_text"> for end users</a>.`

Reply:
Spaces should be correct now!